.changeset/nice-fireants-worry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hopper-ui/components": patch
+---
+
```

Comment:
Some triggers seems not to work, such as Avatar, Label. Link works, so is Button, this is odd.